src/Php54/Php54.php

```diff
@@ -39,4 +39,67 @@ public static function hex2bin($data)
 
         return $data;
     }
+
+    public static function http_response_code($responseCode)
+    {
+        static $currentCode = 200;
+
+        $messages = array(
```

Comment:
> php source already embeds the list for you

No it does.  Apache/Nginx ignores your message and inserts its own.  To see tha native PHP behaviour, runr PHP as a server (`php -S ... -t ...`) and run these scripts:

`header('HTTP/1.0 200 Whatever');`
`header('HTTP/1.0 200');`

In your brower, you'll see these exact headers.  PHP doesn't add "OK".